tutorials/javaFxPart1.md

```diff
@@ -15,6 +15,13 @@ Imagine yourself as a director of a play. First you provision the props that you
 
 A JavaFX application is like a play you are directing. Instead of creating props, you create `Node`s (`Node`s are the fundamental building blocks of a JavaFX application), and place them onto a `Scene` (a scene is a graph of `Node`s). Then, you set your `Scene` on a `Stage` provided by JavaFX. When you call `Stage#show()` method, JavaFX renders a window with your `Stage` on it.
 
+<box type="tip" seamless>
+The `Stage` is like a window in a desktop application. It is the top-level container for a JavaFX application. 
```

Comment:
Maybe we could add, for greater clarity, that the Stage consists of different scenes at different times, i.e. depending on the current action/state, the Scene on the Stage is different?

Reply:
Great point! Will add in the next commit. Thank you!